lib/Quantization/Quantization.cpp

```diff
@@ -360,6 +360,19 @@ static Node *quantizeNode(Function *F, Node *node,
                                     quantizedInputs[1]);
     break;
   }
+  case Kinded::Kind::TileNodeKind: {
```

Comment:
could you add Tile node to quantization e2e to verify that the conversion works properly?